site/docs/concepts/derivations.md

```diff
@@ -30,307 +29,6 @@ Today, Flow enables you to write derivations using either
 [SQLite](#sqlite) or [TypeScript](#typescript).
 Additional language support is in the works.
 
```

Comment:
What about a visible link here for " if you'd like a tutorial introduction to derivations, see ..." etc to let users self-select whether they want this reference-y document or the acmeBank examples.

Reply:
Good idea, added - thanks!